tests/ternary-object-expressions/with-balanced-formatting/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -60,13 +60,13 @@ let isSpace = false;
 
 const dress = isSpace
     ? {
-        spaceSuit: 3,
-        oxygenCylinders: 6
-    }
+          spaceSuit: 3,
+          oxygenCylinders: 6
+      }
     : {
-        shirts: 3,
-        paints: 3
-    };
+          shirts: 3,
+          paints: 3
+      };
```

Comment:
From a closer look at some snapshot changes at this point with update from PR #492, with `tabWidth: 4`:

```diff
@@ -60,13 +60,13 @@ let isSpace = false;
 
 const dress = isSpace
     ? {
-        spaceSuit: 3,
-        oxygenCylinders: 6
-    }
+          spaceSuit: 3,
+          oxygenCylinders: 6
+      }
     : {
-        shirts: 3,
-        paints: 3
-    };
+          shirts: 3,
+          paints: 3
+      };
 
 ================================================================================
 `;
```

The alignment of the inner object members in this diff felt little off to me with my magnified terminal font. I think this is because the inner object members have an indentation of 10 spaces, while most of the other lines have an indentation of 4 or 6 spaces.

I am now thinking that it would be best to use an option to remove the ternary indenting, like "--no-ternary-inner-indent" as discussed in #585 in case of tab width > 2 spaces or `--use-tabs` option.